components/components.css

```diff
@@ -0,0 +1,856 @@
+  font-size: 20px;
+  align-self: center;
+  color: #7cb972;
+  width: 100%;
+}
+
+/* badges classes ------------------------------------------------------------------------------------------------------------ */
+
+.badge {
+  padding: 1px 4px 1px 4px;
+  border-radius: 2px;
+}
+
+.badge-text {
+  font-family: var(--secondary-font);
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 16px;
+}
+
+.bdg-gray {
+  background-color: #b2bec3;
+}
+
+.badge-text-gray {
+  color: rgb(105, 106, 107);
+}
+
+.bdg-green {
+  background-color: #87d37c;
+}
+
+.badge-text-green {
+  color: #2d5c26;
+}
+
+.bdg-red {
+  background-color: #e74c3c;
+}
+
+.badge-text-red {
+  color: #7c2a21;
+}
+
+.bdg-purple {
+  background-color: #be90d4;
+}
+
+.badge-text-purple {
+  color: #5b3c68;
+}
+
+/* cards classes --------------------------------------------------------------------------------------------- */
+
+.card {
+  border: 0.3px solid #e0e0e0;
+  max-width: 250px;
+  transition: all 0.2s ease;
+}
+
+.card:hover {
+  -webkit-box-shadow: 0px -8px 20px -2px rgba(0, 0, 0, 0.14);
+  -moz-box-shadow: 0px -8px 20px -2px rgba(0, 0, 0, 0.14);
+  box-shadow: 0px -8px 20px -2px rgba(0, 0, 0, 0.14);
+}
+
+.card-img {
+  max-width: 100%;
+  margin-bottom: 15px;
+  cursor: pointer;
+}
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  padding: 0 10px;
+  margin-bottom: 10px;
+}
+
+.card-title {
+  font-weight: 600;
+  font-size: 15px;
+  margin-bottom: 5px;
+}
+
+.card-des {
+  font-weight: 400;
+  font-size: 15px;
+  margin-bottom: 15px;
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 600;
+}
+
+.card-heart-icon {
+  width: 25px;
+  height: 25px;
+  cursor: pointer;
+}
+
+.card button {
+  width: 100%;
+}
+
+/* Checkout card */
+
+.card-checkout {
+  background: #f1f1f1;
+  display: flex;
+  color: var(--black);
+}
+
+.card-checkout-img {
+  max-width: 130px;
+  height: 156px;
+}
+
+.card-checkout-info {
+  padding: 10px 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card-checkout-title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 15px;
+}
+
+.title-checkout {
+  margin-bottom: 0;
+}
+
+.card-cross-icon {
+  width: 25px;
+  height: 25px;
+  color: var(--black);
+  cursor: pointer;
+}
+
+.checkout-des {
+  margin-bottom: 20px;
+}
+
+.checkout-price {
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+.card-product-size {
+  display: flex;
+  align-items: center;
+}
+
+.size-text {
+  font-weight: 500;
+  margin-right: 10px;
+}
+
+.p-size {
+  font-size: 14px;
+  font-weight: 500;
+  padding: 5px 10px;
+  border: 0.2px solid var(--black-secondary);
+  border-radius: 50%;
+  margin-right: 5px;
+  cursor: pointer;
+  background: transparent;
+  outline: none;
+}
+
+/* Avatar classes -------------------------------------------------------------------------------------------------- */
+
+.avatar-sm {
+  border-radius: 70px;
```

Comment:
This can be optimized by making another class called avatar and putting the common properties from sm, md, lg in the avatar class. Here you are defining the same border-radius and border in each of them

Reply:
Nice catch ! Will better optimize this.